src/main/webapp/js/userMapData.json

```diff
@@ -846,6 +876,7 @@
       "Charlotte School of Law",
       "Chamberlain College of Nursing",
       "Champlain College",
+      "Charity Guild of Catholic Women",
```

Comment:
Does not seem to be an educational institute, although this could be a valid use case regardless.

Tagging @damithc for advice here.